### DISCUSSION TITLE
Display clock and icon

### DISCUSSION THREAD

**Discussion by marcinmaruszewski** [pixelit-project/PixelIt#267] [Q&A]
Hi, is there a way to display a clock and an icon or two on both sides of the screen? On this page https://www.bastelbunker.de/pixel-it/ I can see that it's possible for a weather card, but what about the clock? 

Maybe there is an API so I can send an Icon and a text message? I have a Home Assistant server, so I'll get current time on my own.

**Comment by foorschtbar** [comment]
Display an icon and text is the main purpose of pixelit. Checkout the documentation how to send data.
